installer/Migration/V5_4_0/Migration.php

```diff
@@ -169,6 +170,47 @@ public function up(): void
         $this->modifyClaimTables();
         $this->modifyClaimRequestCurrencyToForeignKey();
         $this->modifyDefaultRequiredPasswordStrength();
+
+        if (!$this->checkClaimExists()) {
+            $this->getConnection()->createQueryBuilder()
+            ->insert('ohrm_module_default_page')
+            ->values(
+                [
+                    'module_id' => ':module_id',
+                    'user_role_id' => ':user_role_id',
+                    'action' => ':action',
+                ]
+            )
+            ->setParameter('module_id', $this->getDataGroupHelper()->getModuleIdByName('claim'))
+            ->setParameter('user_role_id', 1)
+            ->setParameter('action', 'claim/viewEvents')
+            ->executeQuery();
+
+            $viewClaimModuleScreenId = $this->getConnection()
+            ->createQueryBuilder()
+            ->select('id')
+            ->from('ohrm_screen')
+            ->where('action_url = :action_url')
+            ->setParameter('action_url', 'ViewClaimModule')
+            ->executeQuery()
+            ->fetchOne();
+
+            $this-> insertMenuItems('Claim', $viewClaimModuleScreenId, null, 1, 1300, 1, '{"icon":"admin"}');
+            $parentIdLevel1 = $this->getConnection()
+                ->createQueryBuilder()
+                ->select('id')
+                ->from('ohrm_menu_item')
+                ->Where('menu_title = :menu_title')
+                ->setParameter('menu_title', 'Claim')
+                ->executeQuery()
+                ->fetchOne();
+            $this->insertMenuItems('Configuration', null, $parentIdLevel1, 2, 100, 1, null);
+            $eventListScreenId = $this->getScreenId('Claim events list');
+            $parentId = $this->getParentId('Configuration', $parentIdLevel1);
+            $this->insertMenuItems('Events', $eventListScreenId, $parentId, 3, 100, 1, null);
+            $expenseTypeScreenId = $this->getScreenId('Claim expense types list');
+            $this->insertMenuItems('Expense types', $expenseTypeScreenId, $parentId, 3, 200, 1, null);
```

Comment:
are these order values similar to 4.x product.

Reply:
yes